consensus/qbft/src/integration-test/java/org/hyperledger/besu/consensus/qbft/support/TestContextBuilder.java

```diff
@@ -532,4 +533,21 @@ private static QbftConfigOptions createGenesisConfig(final boolean useValidatorC
     }
     return qbftConfigOptions;
   }
+
+  private static BftValidatorOverrides convertBftForks(final List<QbftFork> bftForks) {
+    final Map<Long, List<Address>> result = new HashMap<>();
+
+    for (final BftFork fork : bftForks) {
```

Comment:
nit: there is probably a cleaner way to do this using streams. but also may not be worth the effort.

Reply:
I actually ripped this straight from https://github.com/hyperledger/besu/blob/a2f517ce32cf8763c858c3e4f051e45c2c56c200/besu/src/main/java/org/hyperledger/besu/controller/QbftBesuControllerBuilder.java#L346-L361

(which now I've looked is actually itself ripped from https://github.com/hyperledger/besu/blob/a2f517ce32cf8763c858c3e4f051e45c2c56c200/besu/src/main/java/org/hyperledger/besu/controller/IbftBesuControllerBuilder.java#L274-L289)

Maybe the original author didn't want too many nested lambdas? Agree it could be neater overall.

I didn't think it was worth removing duplication for the sake of including in the test, but now it's in three places...maybe...thoughts @jframe?

Reply:
not worth the effort, think it's fine to stay as is